test/karma/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "object-shorthand": [2, "never"]
```

Comment:
This actually runs object-shorthand and ensures we don't use shorthand (as opposed to just disabling).